tests/test_dataserializer.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of Grid Singularity Exchange.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from typing import Dict
+from gsy_framework.data_serializer import DataSerializer
+
+
+class TestCompressDecompressDict:
+    """Test compress and decompress dictionaries"""
+
+    @staticmethod
+    def test_compress() -> bytes:
```

Comment:
Similar thing here: since we're testing the `encode_and_compress_dict`, I'd rename this as
```suggestion
    def test_encode_and_compress_dict():
```
I've also removed the `-> bytes` part. The `->` is used to document what type of output is returned by the method. In this case, the test method isn't returning bytes, but rather `None`.